opendj-packages/opendj-docker/run.sh

```diff
@@ -49,17 +50,13 @@ if [ -d "${SECRET_VOLUME}" ]; then
   cp -f ${SECRET_VOLUME}/key* ${SECRET_VOLUME}/trust* ./data/config 2>/dev/null
 fi
 
-# todo: Check /opt/opendj/data/config/buildinfo
-# Run upgrade if the server is older
-
+# Opendj is probably already started in detach mode at the install
 if (bin/status -n | grep Started); then
   echo "OpenDJ is started"
-  # We can't exit because we are pid 1
-  while true; do sleep 100000; done
+  
+  # Use tail instead of sleep to allow the container to be stopped with SIGTERM
+  tail -f /dev/null
 fi
 
-echo "Try to upgrade OpenDJ"
-sh ./upgrade -n
-
```

Comment:
We do not need that since the container is new and has just been updated. It can't be anything else than the container's version.